blivet/partitioning.py

```diff
@@ -2332,8 +2332,11 @@ def growLVM(storage):
         # now that we have grown all thin pools (if any), let's calculate and
         # set their metadata size if not told otherwise
         for pool in vg.thinpools:
-            if not pool.exists and pool.metaDataSize == Size(0):
-                pool.autoset_md_size()
+            if not pool.exists:
+                if pool.metaDataSize == Size(0):
+                    pool.autoset_md_size()
+                elif pool.req_grow:
+                    pool.size -= 2 * pool.metaDataSize
```

Comment:
So this is for the case where metadata size set in kickstart and the pool is growable? Isn't the metadata part reserved in the VG already in that case? That would mean the above should be ``pool.size -= pool.metaDataSize`` to reserve the space for pmspare. Also, the VG should report the missing space for the pmspare on its own, right? (by its ``freeSpace`` property having negative value).